scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -77,6 +86,14 @@ namespace <%= namespace %> {
             });
         }
 
+        public void StartWebCheckout(CheckoutSuccessCallback success, CheckoutFailureCallback failure, CheckoutCancelCallback cancelled) {
```

Comment:
I feel like this method should be called something like `CheckoutWithNativeWebView` or if we want to be inline with naming with Apple Pay checkout `CompleteNativeWebViewCheckout`.

This being called `Start*` I feel like there should be a `FinishWebCheckout` or `FinalizeWebCheckout`.